GUI/_CartPoleGUI_MainWindow.py

```diff
@@ -1062,7 +1063,7 @@ def RadioButtons_simulator_mode(self):
             self.threadpool.start(worker)
 
         else:
-            self.PhysicalCartPoleDriverInstance.terminate_experiment = True
+            self.terminate_experiment = True
```

Comment:
@marcinpaluch1994 Not sure what should go here...can you help me?

Reply:
If you are driving the physical cartpole and opt out in GUI from doing it, you need to terminate the experiment correctly. Don't remember what happens otherwise, but something bad